src/Collection/CommitteeCollection.php

```diff
@@ -24,10 +24,16 @@ public function getOrderedCommittees(CommitteeMembershipCollection $memberships)
             return in_array((string) $committee->getUuid(), $uuids);
         });
 
+        $uuids = $memberships->getCommitteeUnApprovedMemberships(CommitteeMembershipCollection::EXCLUDE_SUPERVISORS)->getCommitteeUuids();
```

Comment:
is the argument useless or not handled in the method ?
`public function getCommitteeUnApprovedMemberships(): self`